src/containers/DiaryHome/DiaryHome.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { Button, Container, TextField } from "@mui/material";
+
+//add css to textField
+const rounded = {
+  borderRadius: "20px",
+  height: "43px",
+  width: "70vw",
+  marginRight: "10px",
+};
+
+function DiaryHome() {
+  //set variables to textfields' values states
+  const [title, setTitle] = useState("");
+  const [message, setMessage] = useState("");
+
+  //onclick listenere to get textfields value when click on submit button
+  function formSubmitHnadler() {
```

Comment:
there is a typo on function name

Reply:
Do you mean the spelling mistake of the function?